lesson32/src/js/register.js

```diff
@@ -1,22 +1,30 @@
 import { showErrorMessage,checkFormValidityInBlur, confirmIfCanSubmit } from "./modules/validation";
+import { toggleInertAttribute } from "./modules/toggle-inert";
 import { togglePasswordDisplay } from "./modules/togglepassword";
 
 const bodyElement = document.querySelector("body");
 const checkboxLink = document.getElementById("js-checkbox-link");
+const modal = document.getElementById("js-modal-area");
 const modalCloseButton = document.getElementById("js-modal-close-button");
 const modalInner = document.getElementById("js-modal-inner");
 const submitButton = document.querySelector(".js-submit-button");
 const invalidItems = document.getElementsByClassName("invalid");
+const focusControlTargets = [document.getElementById("js-form"), document.getElementById("js-header")];
```

Comment:
IMO
Considering where they are used, I felt it was easier to understand them one at a time rather than putting them together in an array here.
```suggestion
const form = document.getElementById("js-form");
const header = document.getElementById("js-header");
```

Reply:
@nakagawamai 
Thank you for your advice!
That's right!
This is easier to understand and has been corrected.
[d081c48](https://github.com/haru-programming/JavaScript-lessons/pull/60/commits/d081c48158664189805e51b4b38131c17fbda099)